src/redisearch_rs/c_entrypoint/inverted_index_ffi/src/lib.rs

```diff
@@ -61,6 +64,27 @@ pub enum InvertedIndex {
     Numeric(EntriesTrackingIndex<Numeric>),
 }
 
+impl Display for InvertedIndex {
```

Comment:
Why do we need the Display trait impl and why do we only ever want the variant name?

Reply:
I could not derive `Debug` since none of the inner types (20+ types) implement debug. So I just manually implemented `Debug` instead to skip the inner type in https://github.com/RediSearch/RediSearch/pull/6837/commits/5ae65e92ab701dc360f480c1e9ff33bee9137f0d